Makefile

```diff
@@ -60,35 +68,24 @@ lint-ui:
 
 lint: lint-ui lint-server ## Run linters
 
-lerna-build: yarn-install
+build:
+	yarn
 	export PATH=$$(pwd)/node_modules/.bin:$$PATH && lerna run build
 
-npm-packages: lerna-build
-	mkdir -p dist
-	$(call PACKAGE_LAB_EXTENSION,application)
-	$(call PACKAGE_LAB_EXTENSION,theme)
-	$(call PACKAGE_LAB_EXTENSION,code-snippet)
-	$(call PACKAGE_LAB_EXTENSION,pipeline-editor)
-	$(call PACKAGE_LAB_EXTENSION,python-runner)
-	cd dist && curl -o jupyterlab-git-0.20.0.tgz $$(npm view @jupyterlab/git@0.20.0 dist.tarball) && cd -
-	cd dist && curl -o jupyterlab-toc-3.0.0.tgz $$(npm view @jupyterlab/toc@3.0.0 dist.tarball) && cd -
-
-bdist: npm-packages
+install-server: ## Build and install backend
 	python setup.py bdist_wheel
-
-install: bdist lint ## Build distribution and install
 	pip install --upgrade dist/elyra-*-py3-none-any.whl
-	$(call UNLINK_LAB_EXTENSION,application)
-	$(call UNLINK_LAB_EXTENSION,theme-extension)
-	$(call UNLINK_LAB_EXTENSION,code-snippet-extension-experimental)
-	$(call UNLINK_LAB_EXTENSION,pipeline-editor-extension)
-	$(call UNLINK_LAB_EXTENSION,python-runner-extension)
-	jupyter lab clean
+
+install-external-extensions:
+	pip install --upgrade jupyterlab-git==0.20.0
+	jupyter labextension install --no-build @jupyterlab/toc@3.0.0
+
+install: lint build install-server install-external-extensions ## Build and install
```

Comment:
I'm wondering if we shouldn't split `install-server` into `build-server` and `install-server` and rename `build` to `build-ui`.  (It doesn't look like there can be separate _build_ and _install_ steps for the UI components, but if there were, then add `install-ui`.)

I also think the `lint` should be removed here and let `build` (or whatever it becomes) depend on `lint-ui` and let `build-server` depend on `lint-server`.  That way, devs working on "either side" will always perform the appropriate linting steps (assuming they build using the makefile).

Reply:
I actually like this. I'll modularize this based on your ideas and push a new commit

Reply:
I just want us to be consistent like we had `install-backend` and `test-server`.